test/rbs/test/runtime_test_test.rb

```diff
@@ -1,14 +1,13 @@
 require "test_helper"
-
 require "rbs/test"
 require "logger"
 
 return unless Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.7.0')
 
-class RBS::Test::RuntimeTestTest < Minitest::Test
+module TestSetupHelper
```

Comment:
I don't think we need to extract this module. Define the helper methods in `RuntimeTestTest`.